src/client/src/apps/MainRoute/widgets/workspace/Workspace.tsx

```diff
@@ -3,8 +3,9 @@ import { TearOff } from 'rt-components'
 import styled from 'styled-components/macro'
 import SpotTileContainer from '../spotTile/SpotTileContainer'
 import { WorkspaceHeader, TileView } from './workspaceHeader'
-import { appendTileViewToUrl } from './utils'
+import { createPopOutUrl } from './utils'
 import { ExternalWindowProps } from './selectors'
+import { useLocalStorage } from '../../../../rt-util/hooks/useLocalStorage'
```

Comment:
Just a minor suggestion - it also works if we import `from 'rt-util/hooks/useLocalStorage'` without the `../../` chain, doing it that way the reference is not lost e.g. if the file is moved or the line is copied and pasted elsewhere.

Also could reexport the hook in `rt-util/index.ts` so it can be used just as `import { useLocalStorage } from 'rt-util'`, which is a pattern already in use when consuming other utility hooks e.g:

![image](https://user-images.githubusercontent.com/4398367/95609784-34d8fb80-0a57-11eb-9422-b42a6b29e3da.png)

Reply:
Makes sense, updated.